tools/pd-tso-bench/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/prometheus/client_golang v1.11.1
 	github.com/tikv/pd/client v0.0.0-00010101000000-000000000000
 	go.uber.org/zap v1.24.0
+	google.golang.org/grpc v1.54.0
```

Comment:
why import this?

Reply:
because this pr forget add it
https://github.com/tikv/pd/pull/6699/files#diff-6e9d6d039becbedf1b1469724f64b7a7c5ecf74a0d9bc24b29ca5e0681443ccdR36